ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/search/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
```

Comment:
- [ ] Was there a requirement for a design dialog?

Reply:
No there was not. Will remove it